waterlog/src/Components/Segments.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { fetchSegments } from '../actions/SegmentActions';
+
+class Segments extends Component {
+    componentWillMount() {
```

Comment:
check your indents. There is plenty of vscode extensions available to allow for formatting of react files that can automatically format your files on save for instance. I would suggest using eslint as it can be setup to do both formatting and checking for good code quality